show/main.py

```diff
@@ -792,97 +841,30 @@ def mac(vlan, port, verbose):
     run_command(cmd, display_cmd=verbose)
 
 #
+# 'show route-map' command ("show route-map") 
+#
+
+@cli.command('route-map')
+@click.argument('route_map_name', required=False)
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+def route_map(route_map_name, verbose):
+    """show route-map"""
+    cmd = 'sudo vtysh -c "show route-map'
+    if route_map_name is not None:
+        cmd += ' {}'.format(route_map_name)
+    cmd += '"'
+    run_command(cmd, display_cmd=verbose)
+#
 # 'ip' group ("show ip ...")
 #
 
 # This group houses IP (i.e., IPv4) commands and subgroups
-@cli.group(cls=AliasedGroup, default_if_no_args=False)
+@cli.group()
 def ip():
     """Show IP (IPv4) commands"""
     pass
 
 
-#
-# get_if_admin_state
-#
-# Given an interface name, return its admin state reported by the kernel.
-#
-def get_if_admin_state(iface):
```

Comment:
Why are you removing these? Was your code out-of-date?

Reply:
My bad. I thought I had grabbed the most recent version of the file but it looks like I missed a step.   I've updated this now.